packages/synapse-interface/components/Portfolio/PortfolioTokenAsset.tsx

```diff
@@ -39,19 +45,62 @@ const handleFocusOnInput = () => {
   inputRef.current.focus()
 }
 
+function checkCCTPChainConditions(
+  fromChainId: number,
+  toChainId: number
+): boolean {
+  switch (true) {
+    case fromChainId === ETH.id && toChainId === ARBITRUM.id:
+      return true
+    case fromChainId === ARBITRUM.id && toChainId === ETH.id:
+      return true
+    case fromChainId === ETH.id && toChainId === AVALANCHE.id:
+      return true
+    case fromChainId === AVALANCHE.id && toChainId === ETH.id:
+      return true
+    case fromChainId === ARBITRUM.id && toChainId === AVALANCHE.id:
+      return true
+    case fromChainId === AVALANCHE.id && toChainId === ARBITRUM.id:
+      return true
+    default:
+      return false
+  }
+}
+
+function checkIfUsingCCTP({
+  fromChainId,
+  fromToken,
+  toChainId,
+  toToken,
+}: {
+  fromChainId: number
+  fromToken: Token
+  toChainId: number
+  toToken: Token
+}): boolean {
+  const originTokenSymbol = fromToken?.symbol
+  const destinationTokenSymbol = toToken?.symbol
+
+  const isTokensUSDC: boolean =
+    originTokenSymbol === 'USDC' && destinationTokenSymbol === 'USDC'
```

Comment:
would prefer to compare by the tokens themselves vs the symbols (since other tokens can have the same symbol' (see `constants/tokens/master` for example):

```
const isTokenUSDC: boolean = fromToken === USDC && toToken === USDC
```

where `USDC` is imported from that constants file

Reply:
ty! need to rem to compare against actual token constant

Reply:
updated in https://github.com/synapsecns/sanguine/pull/1150/commits/51945d365145fde83269a0bd75bd70ee18189e12